source/default/_patterns/00-protons/legacy/css/global.css

```diff
@@ -247,3 +260,79 @@ div.site-alert :where(div.severity-medium,div.severity-high,div.severity-low) .t
 .block--search form input[type='submit']:hover {
   @apply text-utc-new-gold-500;
 }
+
+/****NEW BRANDING SET-UP as of March 18, 2022*****/
+
+:root {
+  --utc-transition: all .4s ease-in-out;
+}
+body {
+  font-size: 18px;
+  @apply font-utcbody text-utc-new-blue-500 text-left bg-utc-bg-page text-base font-light m-0 overflow-x-hidden overflow-y-scroll;
+  animation: fadeIn 2s;
+}
+@keyframes fadeIn {
+  0% {opacity:0;}
+  100% {opacity:1;}
+}
+a {
+  @apply text-utc-links-static outline-none;
+  transition:  var(--utc-transition);
+}
+.department-info-- .social-media-links--platforms a {
+  @apply text-utc-new-blue-500;
+}
+a:hover,
+a:focus,
+.department-info-- .social-media-links--platforms a:hover,
+.department-info-- .social-media-links--platforms a:focus {
+  @apply text-utc-new-gold-500 outline-none;
+}
+span:target {
+  @apply block relative invisible -top-2.5;
+}
+p {
+  @apply text-gray-800 m-0 mb-4 p-0 text-utcbody;
+}
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+@apply font-bold text-utc-new-blue-500 font-utcheadings;
+}
+b, strong {
+  @apply font-medium;
+}
+blockquote, dd, dl, figure, h1, h2, h3, h4, h5, h6, hr, p, pre {
+  @apply ml-0;
+}
+p {
+  @apply mb-5;
+}
+.UTCtextblock__links p {
+  text-indent:24px;
+}
+.UTCtextblock__links hr {
+  @apply my-6;
+}
+.UTCtextblock__links h2 {
+  @apply mb-5;
+}
+.UTCtextblock__links h2:first-child {
+  @apply mt-0;
+}
+.department-info-- .field--name-field-utc-department-hours .field__label {@apply font-utcheadings mt-8 mb-4}
+.utc-hero-block a {@apply text-utc-new-blue-500;}
+.utc-hero-block a:hover {@apply text-utc-new-gold-500;
+@media screen and (max-width: 992px) {
```

Comment:
It may make sense to stay in between
https://v2.tailwindcss.com/docs/container.
But if there's a way we could use those properties that be great.